core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeImportService.java

```diff
@@ -0,0 +1,100 @@
+package bio.terra.pearl.core.service.export;
+
+import bio.terra.pearl.core.model.audit.DataAuditInfo;
+import bio.terra.pearl.core.model.participant.Enrollee;
+import bio.terra.pearl.core.model.participant.ParticipantUser;
+import bio.terra.pearl.core.model.participant.Profile;
+import bio.terra.pearl.core.model.study.StudyEnvironment;
+import bio.terra.pearl.core.model.workflow.HubResponse;
+import bio.terra.pearl.core.service.exception.internal.InternalServerException;
+import bio.terra.pearl.core.service.export.formatters.module.EnrolleeFormatter;
+import bio.terra.pearl.core.service.export.formatters.module.ParticipantUserFormatter;
+import bio.terra.pearl.core.service.participant.ProfileService;
+import bio.terra.pearl.core.service.workflow.EnrollmentService;
+import bio.terra.pearl.core.service.workflow.RegistrationService;
+import com.opencsv.CSVParser;
+import com.opencsv.CSVParserBuilder;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvValidationException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.*;
+
+@Service
+public class EnrolleeImportService {
+    /** for now, we only support importing from a specific style of export. */
+    ExportOptions IMPORT_OPTIONS = new ExportOptions(false, true, true, ExportFileFormat.TSV, null);
```

Comment:
Nit: should we rename `ExportOptions` if it's also being used for imports? I could also see the argument that we are importing data exported with these `ExportOptions`... Maybe `SerializationOptions`?

Reply:
that's a good idea -- I'll see if these diverge or not over upcoming PRs, and if they stay together, I like "serializationOptions" as the name